opal/mca/common/ofi/common_ofi.c

```diff
@@ -391,9 +399,13 @@ int opal_common_ofi_mca_register(const mca_base_component_t *component)
             ret = verbose_index;
             goto err;
         }
+    } else {
+        verbose_index = param;
     }
 
-    if (0 > accelerator_rank_index) {
+
+    param = mca_base_var_find("opal", "opal_common", "ofi", "accelerator_rank");
```

Comment:
I think this looks good, there is a case that could be made that this should not be an ofi parameter, but an mca parameter of the accelerator framework (for a later PR)

Reply:
Ah good point.  I was just keeping things the way they were.  I've no objection to moving it in the end to accel framework.